share/mediation_integrate.F

```diff
@@ -1701,6 +1701,8 @@ SUBROUTINE open_hist_w ( grid , config_flags, stream, alarm_id, &
    ENDIF
 
    ierr = 0
+   fname = ""
+   n2 = ""
 
```

Comment:
this change looks irrelevant to the PR

Reply:
@dudhia, True, but it is also needed because it triggers an error in some strict compilers. The Fortran standard dictates that every **INTENT(OUT)** variable needs to be initialized. Putting a PR for adding two lines to the code that does not affect any of the computations is somewhat absurd. Otherwise, it will be ignored during updating. However, we have a lot of users worldwide that will make a fuzz for a simple error like that. We intend to use WRF as it is distributed in this repository without change. We are trying to avoid patching WRF every time there is a release. Many users have WRF-ROMS coupled applications using the ESMF/NUOPC or MCT coupling frameworks. 

We separated our PR contribution (https://github.com/wrf-model/WRF/pull/1923) to compile WRF with the NetCDF-4 library. Are you suggesting that to make this correction, we need to separate adding these inconsequential but need to lines to **mediation_integrate.F**?

Reply:
OK, I was just making sure it was unrelated but needed. Since it is so small it can be combined in this PR. It was described in the PR I now see.